src/contexts/PickupOrDeliveryContext.java

```diff
@@ -0,0 +1,31 @@
+package contexts;
```

Comment:
remove this file and transition to application context.

Also change from boolean to enum